.vscode/launch.json

```diff
@@ -25,6 +25,18 @@
       "console": "internalConsole",
       "internalConsoleOptions": "openOnSessionStart",
       "outputCapture": "std"
+    },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Debug current test",
```

Comment:
What if the current editor is not a test file?

Reply:
Then the debugger will quickly finish with `0 passing` if it cannot find test cases.
I am not aware of an easy way to restrict this, as the variable simply delivers the currently opened file (see also https://code.visualstudio.com/docs/editor/variables-reference#_predefined-variables).